datasets/goes/goes-glm/dataset.yaml

```diff
@@ -22,15 +23,20 @@ collections:
         token: ${{ pc.get_token(goeseuwest, noaa-goes16) }}
         chunks:
           options:
-            name_starts_with: GLM-L2-LCFA/
             ends_with: .nc
-            chunk_length: 30
+            chunk_length: 10000
```

Comment:
Just an FYI @pjhartzell, we needed to adjust these chunking values when running it on the full thing. pctasks was struggling with this many small chunkfiles. Rob made a few change that'll help, but we should get better overall throughput with fewer, bigger chunks.